flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -183,9 +221,15 @@ function start_cluster {
   "$FLINK_DIR"/bin/start-cluster.sh
 
   # wait at most 10 seconds until the dispatcher is up
+  local QUERY_URL
+  if [ "x$USE_SSL" = "xON" ]; then
```

Comment:
why the `x`?

Reply:
A guard against empty `USE_SSL` - just tested without and it seems working (probably because of the quotes) but I would not bet that this is the behaviour in every shell...
If it was empty, and a shell executes this as `if [ $USE_SSL = ON ] ; then echo "ON"; fi` then you'll get something like this: `bash: [: =: unary operator expected`